### ISSUE TITLE
Pipeline jobs - Granurality

### ISSUE THREAD

**Issue by ChadiEM** [jan-molak/jenkins-build-monitor-plugin#249]
Hello. A nice addition would be to show more details about the currently executing stages of a Jenkins 2 pipeline job, instead of just showing the job running.

**Comment by jan-molak** [comment]
I agree, indeed it would! I'd be happy to assist with pull requests if anyone would like to pick this one up?

**Comment by ChadiEM** [comment]
I started working on this. For now, only displays the current stage(s). Preview below:

![preview](https://cloud.githubusercontent.com/assets/1502811/22174365/39f07754-dfdd-11e6-9c41-3c23c3f087d6.png)

Feedback is welcome!

**Comment by jan-molak** [comment]
@ChadiEM - awesome! It's great that you picked it up :-)
